descriptors/ModuleDescriptor-template.json

```diff
@@ -1985,7 +2032,8 @@
         "inventory-storage.items.item.put",
         "scheduled-notice-storage.scheduled-notices.collection.delete",
         "scheduled-notice-storage.scheduled-notices.item.post",
-        "users.item.get"
+        "users.item.get",
+        "pubsub.publish.post"
```

Comment:
The reference UI does not currently use this endpoint for manual change due date, it edits the loan directly using a `PUT /circulation/loans/{id}`, that endpoint will need similar logic and permissions

Reply:
Added publishing logic to the `LoanCollectionResource.replace`, a test for it and PubSub permission to `modperms.circulation.loans.item.put`.